app/scripts/modules/core/src/search/global/GlobalSearchResults.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { UISref } from '@uirouter/react';
+
+import { SETTINGS } from 'core/config/settings';
+import { ISearchResult } from 'core/search/search.service';
+import { ISearchResultSet } from 'core/search/infrastructure/infrastructureSearch.service';
+import { SearchResult } from 'core/search/infrastructure/SearchResult';
+
+export interface IGlobalSearchResultsProps {
+  categories: ISearchResultSet[];
+  query: string;
+  onItemKeyDown: (event: React.KeyboardEvent<HTMLAnchorElement>) => any;
+  onResultClick: (result: ISearchResult) => any;
+  onSeeMoreClick: Function;
+  resultRef: (categoryIndex: number, resultIndex: number, ref: HTMLAnchorElement) => any;
+  seeMoreRef: (ref: HTMLAnchorElement) => any;
+}
+
+export function GlobalSearchResults({
+  categories,
+  query,
+  onItemKeyDown,
+  onResultClick,
+  onSeeMoreClick,
+  resultRef,
+  seeMoreRef
+}: IGlobalSearchResultsProps) {
+  const { searchVersion } = SETTINGS;
+
+  if (!categories.length) {
+    return (
+      <ul className="dropdown-menu" role="menu">
+        <li className="result">
+          <a>No matches</a>
+        </li>
+      </ul>
+    );
+  }
+
+  return (
+    <ul className="dropdown-menu" role="menu">
+      {categories.map((category, categoryIndex) => ([
+        <li key={category.type.id} className="category-heading">
+          <div className="category-heading">{category.type.displayName}</div>
+        </li>,
+        category.results.map((result, index) => (
+          <li key={result.id} className="result">
+            <a
```

Comment:
seems to behave like a button. We can make this a <button className="link"> if we want it to look like a link.

Reply:
re: both this and the other comment on `<a>` tags that have `href`s, what's our story on moving things to `<button>`s (which for the record I am _all about_) when dealing with either a raw `href` OR dealing with an anchor wrapped by a `<UISref>`?

- For the <UISref> ones, is it safe to use a `<button>` as the child instead of an `<a>`? It kinda looks like it based on [the onClick handler `UISref` adds to it's children](https://github.com/ui-router/react/blob/22b1413/src/components/UISref.tsx#L74), but it's also adding an `href` 🤷‍♂️.

- For the ones that use raw `href`s currently, do we have some kind of ui-router-y thing to do navigation to URLs rather than ui-router states? When I was talking to @christopherthielen about this he made it seem like ui-router doesn't like full URLs and I should just avoid touching them for now.

cc @christopherthielen

Reply:
`UISref` adds an href to everything in `@uirouter/react` but I think that's a bug.  I think it should only add href to anchor tags. 
 It adds the onClick handler regardless

For raw hrefs, it works fine because they are "hash" links, i.e. `"#/route/to/some/thing" and we can handle the hashchange event.  There are other reasons to use UISref instead of links, such as it "inherits" (keeps) all current parameter values when activating the linked state.

Reply:
@christopherthielen to be clear though, you're not recommending I try to use the raw `href` urls I have coming back from the search service with `UISref`, right? When we talked about this previously I got the impression I should stick with raw `href`s  until we go back and refactor the shape of those result objects to have some router-compatible state + params.

Reply:
not recommending `UISref` *AND* manual `href`.  For these raw `hrefs` use a standard anchor tag.

Reply:
cool, just making sure 😄